src/components/Author/__tests__/__snapshots__/author.test.tsx.snap

```diff
@@ -38,23 +14,7 @@ exports[`Author component renders correctly 1`] = `
   >
     <img
       alt="test-author"
-      className="author-img"
-      css={
-        Object {
-          "map": undefined,
-          "name": "bq2cf",
-          "next": undefined,
-          "styles": "
-  height: 30px;
-  width: 30px;
-  margin-top: 5px;
-  border-radius: 100%;
-  border: 1px solid var(--brand-light);
-  transition: all 0.2s ease-in-out;
-",
-          "toString": [Function],
-        }
-      }
+      className="img "
```

Comment:
I think this is a good move to standardize to SCSS. I just wanted to check - all the class name changes to 'img' have a space after the 'g'. Was this intentional?